Alexandr Antsiferov/lesson8/src/app/components/UsersList.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import User from "./User";
+
+const UsersList = props => {
+  const [users, setUsers] = useState({});
```

Comment:
```suggestion
  const [users, setUsers] = useState([]);
```
скорее массив, а не объект:)